packages/bpk-component-image/src/BpkImage-test.tsx

```diff
@@ -43,7 +43,7 @@ describe('BpkImage', () => {
       <BpkImage
         altText="image description"
         aspectRatio={816 / 544}
-        style={{ width: spacingSm }}
+        style={{ width: spacings.onePixelRem }}
```

Comment:
Could we just put a bigger value in here? Concern here is that the tests may not catch any issues if its only 1rem in width

Reply:
This test isn't visual, only testing that user values are placed onto the correct area of DOM, I can absolutely increase the value, however, it will provide no increase to the validity of the test.